otter/integration/tests/test_convergence.py

```diff
@@ -183,6 +273,25 @@ def _choose_half_the_servers(self, (code, response)):
         self.n_killed = self.n_servers / 2
         return ids[:self.n_killed]
 
+    def _choose_servers_from_active_list(self, (code, response),
```

Comment:
Don't use (code, response) here.  Write this like:

```
def _choose_servers_from_active_list(self, state, num_servers=None):
    code, response = state
    ...rest of function here...
```

This has to do with Python 3 upward compatibility.  Because Python 3 is poopy-pants.